foolbox/models/pytorch.py

```diff
@@ -108,14 +105,10 @@ def predictions_and_gradient(self, image, label):
         input_shape = image.shape
         image, dpdx = self._process_input(image)
         target = np.array([label])
-        target = torch.from_numpy(target)
-        if self.cuda:  # pragma: no cover
-            target = target.cuda()
+        target = torch.from_numpy(target).to(self.device)
 
         images = image[np.newaxis]
-        images = torch.from_numpy(images)
-        if self.cuda:  # pragma: no cover
-            images = images.cuda()
+        images = torch.from_numpy(images).to(self.device)
 
         if self._old_pytorch():  # pragma: no cover
```

Comment:
By using .to(device) we are anyway breaking compatibility with older pytorch versions in which case we can remove these checks and accompanying fallbacks altogether.